notifications_utils/template.py

```diff
@@ -655,6 +655,8 @@ def subject(self):
 class BaseLetterTemplate(SubjectMixin, Template):
 
     template_type = "letter"
+    max_page_count = LETTER_MAX_PAGE_COUNT
+    max_sheet_count = max_page_count // 2
```

Comment:
```suggestion
    max_sheet_count = LETTER_MAX_PAGE_COUNT // 2
```

This reads a bit funny as-is; feels like it implies if you override `max_page_count` in a subclass then `max_sheet_count` will respect that change, but it won't. So probably better to derive from the external constant or turn it into a `@property` - don't mind which.

Reply:
Changed in 0d50d5e

>    **Derive sheet count from constant**
>
>    This reduces the implication that overriding `max_page_count` on an
>    instance will also update the sheet count.
>
>    Prefer this to implementing an `@property` because this way it’s still
>    accessible on both the class and any instances.